docs/guides/hedgehog-engine/rangers/cyloop/color.md

```diff
@@ -0,0 +1,73 @@
+---
+description: Cyloop color replacement guide for Sonic Frontiers
+---
+# Replacing Cyloop Color
+
+!!! info
+    Requirements:
+    - Playercommon.pac
+    - player_common.rfl
+    - Hedgearcpack
+    - [010 editor](https://www.sweetscape.com/010editor/)
```

Comment:
Guide links should be followed by `{ target="_blank" }`, in order to make them open in a new tab